src/lib/coerce.js

```diff
@@ -44,6 +44,20 @@ exports.valObjects = {
             if(opts.coerceNumber) v = +v;
             if(opts.values.indexOf(v) === -1) propOut.set(dflt);
             else propOut.set(v);
+        },
+        validateFunction: function(v, opts) {
```

Comment:
oh and I should add, I didn't want to add regex support in the (regular) `coerceFunction` defined above for performance reasons.